packages/authentication/services/auth.ts

```diff
@@ -10,8 +10,11 @@ import type {
 } from '../types/auth'
```

Comment:
You shoudln't be here, the old endpoints were removed from here. AllAuth endpoints are in the AllAuthApi class. Did you base this off the correct branch? BA-1786-fe-mfa-setup-fe-library